source/module_hsolver/kernels/test/perf_math_kernel.cpp

```diff
@@ -0,0 +1,108 @@
+#include "module_base/blas_connector.h"
+#include "module_base/constants.h"
+#include "module_hsolver/kernels/math_kernel_op.h"
+#include "module_psi/kernels/memory_op.h"
+
+#include <complex>
+#include <benchmark/benchmark.h>
+#include <iostream>
+
+class PerfModuleHsolverMathKernel : public benchmark::Fixture {
+    public:
+
+    // DEVICE SYMBOL
+    const psi::DEVICE_CPU* cpu_ctx = {};
+    const psi::DEVICE_GPU* gpu_ctx = {};
+
+    int dim_vector = 1;
+
+    std::complex<double>* test_zvector_a,* test_zvector_b,* result_zvector;
+
+    double* test_dvector_a;
+
+    double constant = 1.24;
+
+    void SetUp(const benchmark::State& state){
+        dim_vector = state.range(0) * state.range(0) * 50000 + 1000; // Generate vectors with different diminsions
+
+        // This should be complex vectors
+        test_zvector_a = new std::complex<double>[dim_vector + 1];
+        test_zvector_b = new std::complex<double>[dim_vector + 1];
+        result_zvector = new std::complex<double>[dim_vector + 1];
+
+        // The following is double vectors
+        test_dvector_a = new double[dim_vector + 1];
+        for (int i=0;i<dim_vector;i++){
+            test_zvector_a[i] = std::complex<double>{152.2451079 * i - 627.3598126 + 1.231609472 * i * i,11.5973122 * i * i * i - 21.3097421 * i * i * i * i};
+            test_zvector_b[i] = std::complex<double>{11.5973122 * i * i * i - 21.3097421 * i * i * i * i,152.2451079 * i - 627.3598126 + 1.231609472 * i * i};
+            test_dvector_a[i] = 241.21451 * i * i * i - 1.9582432 * i * i * i * i;
+        }
+        constant = state.range(0) * state.range(0) - 662.96725;
+    }
+    void TearDown(const benchmark::State& state){
+        delete[] test_zvector_a;
+        delete[] test_zvector_b;
+    }
+
+
+    // OPs need benchmark
+    // CPU operator
+    using zdot_real_cpu_op = hsolver::dot_real_op<std::complex<double>, psi::DEVICE_CPU>;
+    using zdot_real_gpu_op = hsolver::dot_real_op<std::complex<double>, psi::DEVICE_GPU>;
+    
+    using vector_div_constant_op_cpu = hsolver::vector_div_constant_op<std::complex<double>, psi::DEVICE_CPU>;
+    using vector_mul_vector_op_cpu = hsolver::vector_mul_vector_op<std::complex<double>, psi::DEVICE_CPU>;
+    using vector_div_vector_op_cpu = hsolver::vector_div_vector_op<std::complex<double>, psi::DEVICE_CPU>;
+    using constantvector_addORsub_constantVector_op_cpu
+        = hsolver::constantvector_addORsub_constantVector_op<std::complex<double>, psi::DEVICE_CPU>;
+    using axpy_op_cpu = hsolver::axpy_op<std::complex<double>, psi::DEVICE_CPU>;
+    using scal_op_cpu = hsolver::scal_op<double, psi::DEVICE_CPU>;
+    using gemv_op_cpu = hsolver::gemv_op<std::complex<double>, psi::DEVICE_CPU>;
```

Comment:
I don't think you need to define such alias. We never use gpu codes here so no need to emphasize cpu here.